src/engine/server/server.cpp

```diff
@@ -1585,9 +1585,9 @@ void CServer::SendServerInfo(const NETADDR *pAddr, int Token, int Type, bool Sen
 	}
 
 	ADD_INT(p, PlayerCount); // num players
-	ADD_INT(p, MaxClients-g_Config.m_SvSpectatorSlots); // max players
+	ADD_INT(p, maximum(MaxClients - g_Config.m_SvSpectatorSlots - g_Config.m_SvReservedSlots, PlayerCount)); // max players
```

Comment:
```suggestion
	ADD_INT(p, maximum(MaxClients - maximum(g_Config.m_SvSpectatorSlots, g_Config.m_SvReservedSlots), PlayerCount)); // max players
```

I think it's this way? Not sure. Drop `sv_spectator_slots`?

Reply:
I'm not sure either, but the way these slots work is weird, imagine a server with 15 reserved slots and 1 spectator slot, now people without the password can join the server and take up the "reserved" slots.

Reply:
Reserved slots only work for joining the server currently, not reserving a spot in the game. So it'd be correct to display 0/1 there, I guess.